homeassistant/components/webostv/media_player.py

```diff
@@ -104,15 +106,12 @@ async def cmd_wrapper(self: _T, *args: _P.args, **kwargs: _P.kwargs) -> None:
             await func(self, *args, **kwargs)
         except WEBOSTV_EXCEPTIONS as exc:
             # If TV is off, we expect calls to fail.
-            if self.state == STATE_OFF:
-                level = logging.INFO
-            else:
-                level = logging.ERROR
             _LOGGER.log(
-                level,
-                "Error calling %s on entity %s: %r",
+                logging.INFO if self.state == STATE_OFF else logging.ERROR,
```

Comment:
Note: This PR doesn't change the logging error level behavior but I am wondering if we should use different logging levels if the device is on or off and if we do should a service call when the device is off logged as info or warning?

Reply:
Do you mean we should just change info for warning level? We already use different logging levels, right? I'm not completely following your question.

Reply:
Yes we already use different logging levels, I asked if we should change it, the options as I see it are:
1. Don't change anything (device off = info level, device on = error level)
2. Change info to warning (device off = warning level, device on = error level)
3. Use error level either if device is on or off.

I asked mainly to understand the recommended behavior in media player entities, there is no functionality change in this PR for the logging.

Reply:
I don't think we have a single standard to handle this. Some integrations turn on the device if it's not on before executing other calls.

Generally we raise a HomeAssistantError in the service handler if we can't execute the api call.

Reply:
Updated PR description for `HomeAssistantError` (device on) and warning log (device off)